packages/apptive_grid_core/lib/network/apptive_grid_client.dart

```diff
@@ -299,14 +287,14 @@ class ApptiveGridClient {
     final requestUri = uri.replace(
       scheme: baseUrl.scheme,
       host: baseUrl.host,
-      queryParameters: Map.from(uri.queryParameters)
-        ..addAll({
-          if (layout != null) 'layout': layout,
-          if (sorting != null)
-            'sorting':
-                jsonEncode(sorting.map((e) => e.toRequestObject()).toList()),
-          if (filter != null) 'filter': jsonEncode(filter.toJson()),
-        }),
```

Comment:
What happens if the custom headers also define a layout? I would suggest letting the custom headers override the layout if set.

Reply:
The layout is define via the query and not the headers.
I decided that the named parameters should have priority over any parameters that might be defined in the uri itself as that is something users of the api would actually rather use than doing it in the api and it would be more confusing if you were to set for example the layout to `key` layout but in the uri it has `field` layout defined so the named argument would be ignored in that case.